--- internal/pipe/gentoo/gentoo.go
+++ internal/pipe/gentoo/gentoo.go
@@ -62,7 +62,7 @@
   exeinto {{ . }}
 {{- end }}
 {{- range .Installs }}
-{{- if .Keywords }}
+{{- if gt (len .Keywords) 1 }}
   if {{ range $i, $k := .Keywords }}{{ if $i }} || {{ end }}use {{ $k }}{{ end }}; then
     {{- if eq .Source .Target }}
     doexe "{{ .Source }}" || die "Failed to install binary"
@@ -218,14 +218,10 @@
	}
	slices.Sort(keywordsList)

-	var instKeywords []string
-	if len(keywordsList) > 1 {
-		instKeywords = keywordsList
-	}
	installs := []installData{
		{
			Source:   cfg.Name,
			Target:   cfg.Name,
-			Keywords: instKeywords,
+			Keywords: keywordsList,
		},
	}
